### PR TITLE
Add affogato recipe

### PR DESCRIPTION
**Pull request by constantinpape**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Steffen-Wolf** [comment]
Hi I am happy to be listed here :)

**Comment by constantinpape** [comment]
@conda-forge/help-c-cpp @conda-forge/help-python-c this PR is ready for review.
It's a c++ package using pybind11 for python bindings.

File: recipes/affogato/build.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+\
+        -DCMAKE_SHARED_LINKER_FLAGS="${LDFLAGS}" \
+        -DCMAKE_EXE_LINKER_FLAGS="${LDFLAGS}" \
+        -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
+        -DCMAKE_CXX_FLAGS_RELEASE="${CXXFLAGS} -O3 -DNDEBUG" \
+        -DCMAKE_CXX_FLAGS_DEBUG="${CXXFLAGS}" \
+\
+        -DBOOST_ROOT=${PREFIX} \
+        -DBUILD_PYTHON=ON \
+\
```
Comment (xhochy) [inline recipes/affogato/build.sh:32]: I would have expected that the following should suffice:
```suggestion
    ${CMAKE_ARGS} \
```

**Comment by constantinpape** [comment]
Thanks for the comments @xhochy. Indeed it works with the simplified CMake arguments.

**Comment by constantinpape** [comment]
Thanks @xhochy !

**Pull request merged**